stock_warehouse_flow/models/stock_warehouse_flow.py

```diff
@@ -0,0 +1,292 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+from odoo.tools.safe_eval import safe_eval
+
+logger = logging.getLogger(__name__)
+
+
+class StockWarehouseFlow(models.Model):
+    _name = "stock.warehouse.flow"
+    _description = "Stock Warehouse Routing Flow"
+
+    name = fields.Char(required=True)
+    active = fields.Boolean(default=True)
```

Comment:
need also a sequence for evaluating the flows in the right order if multiple are valid